src/main/resources/application.properties

```diff
@@ -1 +1,5 @@
 
+server.port=8081
+spring.data.mongodb.database=User
```

Comment:
you can use spring.data.mongodb.uri=mongodb://localhost:27017/User as well 
mongodb://<host>:<port>/<database> : to specify mongoDB connection string